protocol-units/sequencing/memseq/src/lib.rs

```diff
@@ -148,14 +148,14 @@ pub mod test {
 		assert!(result.is_err());
 		assert_eq!(
 			result.unwrap_err().to_string(),
-			"MempoolTransactionOperationsError error: Mock error: add_transaction"
+			"MempoolTransactionOperationsError error: Other error: add_transaction"
```

Comment:
Should we eventually have a success path test as well?
I still don't see the added value of the `Other` error variant if it can only be produced by the mock.

Reply:
"Other" can be used by a consumer of the trait if it falls outside of the variants of the error enum.